app/models/stream_events/classification_event.rb

```diff
@@ -13,6 +13,11 @@ def process
 
       cache_linked_models!
 
+      if linked_workflow["nero_config"].keys.include? "webhooks"
```

Comment:
that makes sense, i'll move it. that way we don't have to make the check every time